scipy/interpolate/interpnd.py

```diff
@@ -0,0 +1,26 @@
+# This file is not meant for public use and will be removed in SciPy v2.0.0.
+# Use the `scipy.interpolate` namespace for importing the functions
+# included below.
+
+from scipy._lib.deprecation import _sub_module_deprecation
+
+
+__all__ = [  # noqa: F822
+    'CloughTocher2DInterpolator',
+    'GradientEstimationWarning',
+    'LinearNDInterpolator',
+    'NDInterpolatorBase',
+    '_ndim_coords_from_arrays',
```

Comment:
Do we need to include this since it was never public?

Reply:
Removed. (I've no opinion and you have one, so done :-)).